GoogleForCreators/web-stories-wp
# Page Layouts: Improve RTL compatibility

## Context



Page Layouts and Dashboard template previews are inconsistent with the editor in RTL mode.

## Summary


Brings Page Layouts and Dashboard Template previews in line with what they look like when opened in the editor.

## Relevant Technical Choices



Testing this highlighted that some text elements at least from templates render differently in the editor in RTL mode than when they are published/previewed. This issue is represented in a new ticket #6068.

There are no Karma tests set up for Page Layouts yet to snapshot this, so I'm deferring tests to the in progress PR: #5945. I'm not sure what would make sense to test here other than snapshot.

## To-do



## User-facing changes



## Testing Instructions



Turn on RTL mode and ensure that the page layouts in the pane are consistent with what appears when you click to apply them to the page.

---



Fixes #5909
